cimgraph/models/distributed_area.py

```diff
@@ -19,542 +19,42 @@ def __post_init__(self):
         self.distributed_areas = []
         self.boundaries = []
 
-    def build_from_area(self):
+    def build_from_area(self) -> None:
         self.get_all_edges(self.container.__class__)
         self.connection.create_distributed_graph(area=self.container, graph=self.graph)
 
 
+    # Build the distributed model from a topology message from GridAPPS-D
+    def build_from_topo_message(self, topology_dict:dict):
+        # Set base as
+        self.container = self.add_jsonld_to_graph(topology_dict)
 
-    # def build_from_topo_message(self, topology_dict: dict):
+        # builder for Feeder, Substation, VoltageLevel, and Bay
+        if isinstance(self.container, self.cim.EquipmentContainer):
+            for addr_eq in topology_dict['AddressableEquipment']:
```

Comment:
Based on the json file I have, this should be : 
for addr_eq in topology_dict['FeederArea']['AddressableEquipment']: